client/src/components/PredictionPanel.vue

```diff
@@ -47,46 +60,83 @@ export default defineComponent({
       ScorecardSummaryMessages,
     };
   },
+  // TODO: Handle error state where there is no lead prediction
+  // after the API has returned.
   computed: {
-    // Predicted estimate of lead.
-    percentLead(): number | null {
+    // Predicted lead likelihood for parcel's public lines.
+    publicLeadLikelihood(): string | null {
+      // Note that low prediction is the same as high prediction in the DB
+      // right now bc we don't yet have intervals
+      return this.formatPercentage(this.leadState?.data?.publicLeadLowPrediction);
+    },
+    // Predicted lead likelihood for parcel's private lines.
+    privateLeadLikelihood(): string | null {
+      // Note that low prediction is the same as high prediction in the DB
+      // right now bc we don't yet have intervals
+      return this.formatPercentage(this.leadState?.data?.privateLeadLowPrediction);
+    },
+    // Predicted estimate of lead for water systems.
+    percentLead(): string | null {
       const leadServiceLines = this.leadState?.data?.leadServiceLines;
       const serviceLines = this.leadState?.data?.serviceLines;
 
       // Protect against dividing by 0.
       if (leadServiceLines != null && serviceLines != null && serviceLines != 0) {
-        return Math.round(leadServiceLines / serviceLines);
+        return this.formatPercentage(leadServiceLines / serviceLines);
       }
-      // TODO: Handle error state where there is no lead prediction
-      // after the API has returned.
       return null;
     },
     pwsId(): string | null {
       // TODO: Handle error state where there is no water system id
       // after the API has returned.
       return this.geoState?.geoids?.pwsId ?? null;
     },
+    showWaterSystemPrediction(): boolean {
+      return this.geoState?.geoids?.geoType != GeoType.address && this.pwsId != null && this.percentLead != null;
+    },
   },
   watch: {
     // Listen for changes to pws id. Once it changes, a new prediction
     // must be fetched.
-    'geoState.geoids.pwsId': function() {
-      if (this.geoState?.geoids?.pwsId != null) {
+    'geoState.geoids': function() {
```

Comment:
was this the fix? i.e. listen on all geo IDs and not just pws id

Reply:
I updated it to just geos so I could dispatch events on both pws id and geotype/lat/long